Library/Formula/ford.rb

```diff
@@ -0,0 +1,64 @@
+class Ford < Formula
+  desc "Automatic documentation generator for modern Fortran programs"
+  homepage "https://github.com/cmacmackin/ford/"
+  url "https://pypi.python.org/packages/source/F/FORD/FORD-3.0.2.tar.gz"
+  sha256 "30844536c73f51fcb078a090741349a3d13de708a3f3f7a6fa722532a97f3268"
+
+  head "https://github.com/cmacmackin/ford.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "toposort" do
+    url "https://pypi.python.org/packages/source/t/toposort/toposort-1.4.tar.gz"
+    sha256 "c190b9d9a9e53ae2835f4d524130147af601fbd63677d19381c65067a80fa903"
+  end
+
+  resource "Markdown" do
+    url "https://pypi.python.org/packages/source/M/Markdown/Markdown-2.6.2.tar.gz"
+    sha256 "ee17d0d7dc091e645dd48302a2e21301cc68f188505c2069d8635f94554170bf"
+  end
+
+  resource "beautifulsoup4" do
+    url "https://pypi.python.org/packages/source/b/beautifulsoup4/beautifulsoup4-4.3.2.tar.gz"
+    sha256 "a2b29bd048ca2fe54a046b29770964738872a9747003a371344a93eedf7ad58e"
+  end
+
+  resource "markdown-include" do
+    url "https://pypi.python.org/packages/source/m/markdown-include/markdown-include-0.5.1.tar.gz"
+    sha256 "72a45461b589489a088753893bc95c5fa5909936186485f4ed55caa57d10250f"
+  end
+
+  resource "MarkupSafe" do
+    url "https://pypi.python.org/packages/source/M/MarkupSafe/MarkupSafe-0.23.tar.gz"
+    sha256 "a4ec1aff59b95a14b45eb2e23761a0179e98319da5a7eb76b56ea8cdc7b871c3"
+  end
+
+  resource "Jinja2" do
+    url "https://pypi.python.org/packages/source/J/Jinja2/Jinja2-2.7.3.tar.gz"
+    sha256 "2e24ac5d004db5714976a04ac0e80c6df6e47e98c354cb2c0d82f8879d4f8fdb"
+  end
+
+  resource "Pygments" do
+    url "https://pypi.python.org/packages/source/P/Pygments/Pygments-2.0.2.tar.gz"
+    sha256 "7320919084e6dac8f4540638a46447a3bd730fca172afc17d2c03eed22cf4f51"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    %w[toposort Markdown beautifulsoup4 markdown-include MarkupSafe Jinja2 Pygments].each do |r|
+      resource(r).stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/ford", "-V"
```

Comment:
@mikemcquaid and @DomT4 sorry for the slow reply, busy travel weekend for me…

I have a question: it seems that the the `HEAD` is installed then my test fails because somehow `’#{bin}/ford’` isn’t being found. HEAD is virtually identical to 3.0.2 so this has more to do with `#{bin}` I suspect. Should I worry about that, and is there a better way to specify the path to ford?

To address your concerns about the test: I could create a simple Fortran program with embedded documentation and then run FORD on that as the test. What do you think of that?